jenkins/ml-models.JenkinsFile

```diff
@@ -0,0 +1,87 @@
+lib = library(identifier: 'jenkins@2.0.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    agent
+    {
+        docker {
+            label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:release-centos7-clients-v2'
+            alwaysPull true
+        }
+    }
+    options {
+        timeout(time: 1, unit: 'HOURS')
+    }
+    parameters {
+        string(
+            name: 'BASE_DOWNLOAD_PATH',
+            description: 'S3 base path to download artifacts from eg:ml-models/huggingface/sentence-transformers/all-distilroberta-v1. DO NOT include the trailing backlash at the end',
+            trim: true
+        )
+        string(
```

Comment:
After version, there are also two types of folder: torch_script & onnx
will that be any issue? As I'm seeing there are two parameters?

Reply:
The entire version folder is downloaded along with sub-folders. So this will include above mentioned scripts.